lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -823,6 +824,23 @@ def _build_block_base_path(block):
             path.append(block.display_name)
         return list(reversed(path))
 
+    @staticmethod
+    def resolve_problem_descendants(course_key, usage_key):
```

Comment:
Since problems don't have decendants:
```suggestion
    def resolve_block_descendants(course_key, usage_key):
```
Also, I found this function `yield_dynamic_block_descendants` is it helpful in this case?